addons/purchase/report/purchase_report.py

```diff
@@ -25,18 +25,15 @@ class PurchaseReport(models.Model):
     ], 'Order Status', readonly=True)
     product_id = fields.Many2one('product.product', 'Product', readonly=True)
     partner_id = fields.Many2one('res.partner', 'Vendor', readonly=True)
-    date_approve = fields.Date('Date Approved', readonly=True)
+    date_approve = fields.Date('Confirmation Date', readonly=True)
```

Comment:
Why do you use a Date fields? On the purchase order it's a Datetime field and the date_order is a datetime on both object PO and report